superchain/Dockerfile

```diff
@@ -206,6 +207,14 @@ RUN apt-key add /tmp/nodesource.asc && rm /tmp/nodesource.asc
   && apt-get -y install nodejs yarn                                                                                     \
   && rm -rf /var/lib/apt/lists/*
 
+# Install SAM CLI
+ARG SAM_CLI_SOURCE="https://github.com/aws/aws-sam-cli/releases/latest/download/aws-sam-cli-linux-x86_64.zip"
+RUN wget -O /tmp/aws-sam-cli-linux-x86_64.zip ${SAM_CLI_SOURCE}                                                         \
```

Comment:
Why no `curl`, which we use for all other downloads? :)

Reply:
I tried but was failing for some reason, let me retry it.